File: gen_scripts/perl/e10.branch.pl
#!/usr/bin/env perl
# get input from user
print "Would you like a toast? [Yes/No]: ";
my $response=<>;               # acquire string input including newline
chomp $response;               # strip newline

# set response string using one line (ternary statement)
#   Test response to a string
$response_str = ($response eq "Yes") ? "That's great!" : "How about a muffin?";

# output the response string
print "$response_str\n";

File: gen_scripts/python2/e10.branch.py
#!/usr/bin/env python
# get input from user
response = raw_input("Would you like a toast? [Yes/No]: ")

# set response string using one line (ternary-like statement)
#   Test response to a string
response_str = "That's great!" if response == "Yes" else "How about a muffin?"

# output the response string
print response_str